### PR TITLE
Added page creation test

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request updates integration tests**

Also:
- ```object``` -> ```credentials``` fix
- moved ```require``` out of ```describe```

### CODE REVIEW

File: aldryn_events/tests/frontend/integration/pages/page.events.crud.js [hunk 1]
```diff
@@ -12,39 +12,52 @@
...
+    saveButton: element(by.css('.submit-row [name="_save"]')),
+    editPageLink: element(by.css('.col1 a')),
+    testLink: element(by.css('.selected a')),
 
-    cmsLogin: function (object) {
+    cmsLogin: function (credentials) {
         // object can contain username and password, if not set it will
         // fallback to 'admin'
-        object = object || {};
+        credentials = credentials ||
```
Comment (FinalAngel) [inline aldryn_events/tests/frontend/integration/pages/page.events.crud.js]: missing var

File: aldryn_events/tests/frontend/integration/pages/page.events.crud.js [hunk 2]
```diff
@@ -12,39 +12,52 @@
...
+            { username: 'admin', password: 'admin' };
 
         eventsPage.usernameInput.clear();
 
         // fill in email field
-        eventsPage.usernameInput.sendKeys(object.username || 'admin')
-            .then(function () {
+        eventsPage.usernameInput.sendKeys(
+            credentials.username || 'admin').then(function () {
```
Comment (vxsx) [inline aldryn_events/tests/frontend/integration/pages/page.events.crud.js]: no need to do `|| 'admin'`, credentials.username is always defined now

File: aldryn_events/tests/frontend/integration/pages/page.events.crud.js [hunk 3]
```diff
@@ -12,39 +12,52 @@
...
             eventsPage.passwordInput.clear();
 
             // fill in password field
-            eventsPage.passwordInput.sendKeys(object.password || 'admin');
+            eventsPage.passwordInput.sendKeys(
+                credentials.password || 'admin');
```
Comment (vxsx) [inline aldryn_events/tests/frontend/integration/pages/page.events.crud.js]: same

File: aldryn_events/tests/frontend/integration/pages/page.events.crud.js (hunk 2 above)
Comment (FinalAngel) [inline aldryn_events/tests/frontend/integration/pages/page.events.crud.js]: no need for the ``|| 'admin'`` anymore

File: aldryn_events/tests/frontend/integration/pages/page.events.crud.js (hunk 3 above)
Comment (FinalAngel) [inline aldryn_events/tests/frontend/integration/pages/page.events.crud.js]: same here

File: aldryn_events/tests/frontend/integration/specs/spec.events.crud.js [hunk 4]
```diff
@@ -26,4 +26,58 @@ describe('Aldryn Events tests: ', function () {
...
+            // switch to default page content
+            browser.switchTo().defaultContent();
+
+            browser.wait(function () {
+                return browser.isElementPresent(eventsPage.testLink);
+            }, eventsPage.mainElementsWaitTime);
+
+            // validate test link text
+            eventsPage.testLink.getText()
+            .then(function (title) {
```
Comment (vxsx) [inline aldryn_events/tests/frontend/integration/specs/spec.events.crud.js]: no need for the line break here, but if you _really_ want to have it - it has to be indented one level to the right

**Comment by vxsx** [comment]
:+1: 

i'm a bit concerned by the selectors being a bit too generic in some places, but that's nothing to worry about in this particular case

**Comment by Marketionist** [comment]
Yep + they can be renamed later on if it will be needed

**Pull request merged**